include/zephyr/bluetooth/conn.h

```diff
@@ -479,6 +479,50 @@ struct bt_conn_le_tx_power {
 	int8_t max_level;
 };
 
+enum bt_conn_le_tx_power_report_reason {
+	/** Local Transmit power changed. */
+	BT_CONN_LE_TX_POWER_REPORT_REASON_LOCAL_CHANGED = 0,
+
+	/** Remote Transmit power changed. */
+	BT_CONN_LE_TX_POWER_REPORT_REASON_REMOTE_CHANGED = 1,
+
+	/** HCI_LE_Read_Remote_Transmit_Power_Level command completed. */
```

Comment:
The public `conn.h` API isn't really tied to HCI in any way (although it might reuse defined values like I mentioned in my other comment), so I'm not sure a reference to a HCI command is appropriate.

Reply:
I can reference `bt_conn_le_get_remote_tx_power_level` instead? Otherwise I can describe it.

Reply:
> I can reference `bt_conn_le_get_remote_tx_power_level` instead? Otherwise I can describe it.

Either way is fine, thanks.

Reply:
I left that comment as is, as I shifted it into hci_types.h (Which I assume you meant in your other comment.).